Userland/Libraries/LibCore/EventLoop.cpp

```diff
@@ -66,6 +66,23 @@ static thread_local Vector<EventLoop&>* s_event_loop_stack;
 static thread_local HashMap<int, NonnullOwnPtr<EventLoopTimer>>* s_timers;
 static thread_local HashTable<Notifier*>* s_notifiers;
 thread_local int EventLoop::s_wake_pipe_fds[2];
+thread_local bool EventLoop::s_wake_pipe_initialized { false };
+
```

Comment:
The commit message says:

> This change utilizes a flag to always initialize the wake pipe if needed.

What do we mean by "if needed" here? It seems to always get initialized(?)

Reply:
If needed means: as soon as the first event loop of a thread gets constructed. I.e. event-loop-free threads won't create unnecessary pipes. I'll put that in the commit message.